[Enhancement] Footer Changes

lyricalcasanova opened the pull request:
# Edited the footer layout
## Description 🗒️

**Changes**
-Removed "Escape from Tarkov Dev tracker" seems inactive and rarely necessary unless this is required I don't see a use for it
-Removed the commented out link to "API Users" page
+Added "Tarkov Monitor" as a resource linked to the github since its not listed anywhere
~Re-ordered the Resources and External Sources to be more stylish

(Reverted)
~Discord: Simplified to just use a shields.io img/button that shows users online count (Still requires widget to be enabled on server)~
~X/Twitter: Also changed to use shields.io img/button to view X/Twitter~
~Removed the " : " in the "Made with love by"~

## Examples 📸

<!-- OPTIONAL: Screenshots with examples for your changes if they are UI related -->
**Before**
![image](https://github.com/the-hideout/tarkov-dev/assets/33468668/a69d673e-acde-4a71-b4b4-41135f886308)
**After**
![image](https://github.com/the-hideout/tarkov-dev/assets/33468668/78db8f01-7d86-43cb-b6b3-2e5ad1f3234c)

## Related Issues 🔗
#914 (Does not fix the issue but its planned)

GrantBirki commented:
.deploy to development

Razzmatazzz commented:
.unlock development

Shebuka reviewed (changes requested):
> Discord: Simplified to just use a shields.io img/button that shows users online count (Still requires widget to be enabled on server)
> X/Twitter: Also changed to use shields.io img/button to view X/Twitter

These are some significant changes and I would like to keep them as they were for now, revert please.

> Removed the " : " in the "Made with love by"

The use of a colon here is justified as it introduces a list of contributors, so please revert it back too :)

lyricalcasanova commented:
> > Discord: Simplified to just use a shields.io img/button that shows users online count (Still requires widget to be enabled on server)
> > X/Twitter: Also changed to use shields.io img/button to view X/Twitter
> 
> These are some significant changes and I would like to keep them as they were for now, revert please.
> 
> > Removed the " : " in the "Made with love by"
> 
> The use of a colon here is justified as it introduces a list of contributors, so please revert it back too :)

Reverted now back to original formatting to comply with your request 👍

lyricalcasanova commented:
I'm confused... the reason for this PR mainly was to fix the issue originally opened #914 unless maybe you know a way to fix the CORS error without removing the iFrame / making sure no one has a default profile picture / avatar and finding a way to fix the discord widget from extending off the page, I am all for it but currently there is no known resolutions hence why I said in the OP above: 
(I did search for others who had this issue but it seems like the widget for discord was made in 2016 and the most recent posts I found were in 2022 with no answers that are similar to this)
> Not sure why the default profile avatars for discord are throwing CORS errors so I assume it's best to just remove it / replace it (Also when hovered it extended past the bottom of the screen)

![image](https://github.com/the-hideout/tarkov-dev/assets/33468668/d33ef2fa-388a-410e-95e2-5a940d0ea665)

Shebuka commented:
> I'm confused... the reason for this PR mainly was to fix the issue originally opened #914 unless maybe you know a way to fix the CORS error without removing the iFrame / making sure no one has a default profile picture / avatar and finding a way to fix the discord widget from extending off the page, I am all for it but currently there is no known resolutions hence why I said in the OP above: (I did search for others who had this issue but it seems like the widget for discord was made in 2016 and the most recent posts I found were in 2022 with no answers that are similar to this)
> 
> > Not sure why the default profile avatars for discord are throwing CORS errors so I assume it's best to just remove it / replace it (Also when hovered it extended past the bottom of the screen)
> 
> ![image](https://private-user-images.githubusercontent.com/33468668/320472220-d33ef2fa-388a-410e-95e2-5a940d0ea665.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI1Nzk3MjcsIm5iZiI6MTcxMjU3OTQyNywicGF0aCI6Ii8zMzQ2ODY2OC8zMjA0NzIyMjAtZDMzZWYyZmEtMzg4YS00MTBlLTk1ZTItNWE5NDBkMGVhNjY1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDA4VDEyMzAyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThlNzU0NjMzOGE5YWZlZmZkYjg1NjA5OGU0OTNjYzVlZmVlMDI5Y2I2ODNjNzE1Y2IxOTJhMmE0OTgwN2QzNGUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.SwDl3roGyosynkMVtptsLJ45ZcR5M1SJjH9wYSpfzx0)

As of 15 October 2023 they are still actively supporting the widget: https://github.com/discord/discord-api-docs/issues/6447 
I sincerely do not understand why you think the widget is bad?

Its expanding and changing fotter height with animation is not ideal, but that can be solved with layout updates, not by removing the widget.

lyricalcasanova commented:
> > I'm confused... the reason for this PR mainly was to fix the issue originally opened #914 unless maybe you know a way to fix the CORS error without removing the iFrame / making sure no one has a default profile picture / avatar and finding a way to fix the discord widget from extending off the page, I am all for it but currently there is no known resolutions hence why I said in the OP above: (I did search for others who had this issue but it seems like the widget for discord was made in 2016 and the most recent posts I found were in 2022 with no answers that are similar to this)
> > > Not sure why the default profile avatars for discord are throwing CORS errors so I assume it's best to just remove it / replace it (Also when hovered it extended past the bottom of the screen)
> > 
> > 
> > ![image](https://private-user-images.githubusercontent.com/33468668/320472220-d33ef2fa-388a-410e-95e2-5a940d0ea665.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI1Nzk3MjcsIm5iZiI6MTcxMjU3OTQyNywicGF0aCI6Ii8zMzQ2ODY2OC8zMjA0NzIyMjAtZDMzZWYyZmEtMzg4YS00MTBlLTk1ZTItNWE5NDBkMGVhNjY1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDA4VDEyMzAyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThlNzU0NjMzOGE5YWZlZmZkYjg1NjA5OGU0OTNjYzVlZmVlMDI5Y2I2ODNjNzE1Y2IxOTJhMmE0OTgwN2QzNGUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.SwDl3roGyosynkMVtptsLJ45ZcR5M1SJjH9wYSpfzx0)
> 
> As of 15 October 2023 they are still actively supporting the widget: [discord/discord-api-docs#6447](https://github.com/discord/discord-api-docs/issues/6447) I sincerely do not understand why you think the widget is bad?
> 
> Its expanding and changing footer height with animation is not ideal, but that can be solved with layout updates, not by removing the widget.

I can give a list of reasons why I don't think the discord widget is good
1. It expands and causes weird issues where users have to scroll awkwardly through a list of users to even see or access the "join server" button and expands the footer to over half of the monitor on desktop.
2. The discord icon does not link to the server, it links directly to discords website which users will most likely click on if they don't scroll.
3. Do users need to see the voice channels and users in the discord on the page before clicking join server?

If you insist on keeping the discord widget I would almost be more excited to create a customized version of it than using the default widget layout with something like the one below

![image](https://github.com/the-hideout/tarkov-dev/assets/33468668/4519a0a3-9ba7-4bf4-b0fb-0e24ab6c907a)

Just because something "works and everything is loading fine" does not mean its the best option.

If you want to keep everything as is that's ok just edit my PR or close it I'll work on a custom widget if it is a must have when I have time.

Blightbuster commented:
My 5 cents: While the current discord widget is the official one, the two negatives @lyricalcasanova outlined above heavily outway that. Id be onboard with either disabling the folding and make the click action redirect to our join link instead of discords website or just using shields.io at that point. Im using shields.io for many of my repos as well and only ever had good experiences with it so far. 

<a href="https://discord.gg/aHZf7aP" style="padding:10px">
 <img src="https://img.shields.io/discord/687549250435153930?label=Discord&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2&style=for-the-badge" />
</a>

Shebuka commented:
> > > I'm confused... the reason for this PR mainly was to fix the issue originally opened #914 unless maybe you know a way to fix the CORS error without removing the iFrame / making sure no one has a default profile picture / avatar and finding a way to fix the discord widget from extending off the page, I am all for it but currently there is no known resolutions hence why I said in the OP above: (I did search for others who had this issue but it seems like the widget for discord was made in 2016 and the most recent posts I found were in 2022 with no answers that are similar to this)
> > > > Not sure why the default profile avatars for discord are throwing CORS errors so I assume it's best to just remove it / replace it (Also when hovered it extended past the bottom of the screen)
> > > 
> > > 
> > 
> > 
> > As of 15 October 2023 they are still actively supporting the widget: [discord/discord-api-docs#6447](https://github.com/discord/discord-api-docs/issues/6447) I sincerely do not understand why you think the widget is bad?
> > Its expanding and changing footer height with animation is not ideal, but that can be solved with layout updates, not by removing the widget.
> 
> I can give a list of reasons why I don't think the discord widget is good
> 
> 1. It expands and causes weird issues where users have to scroll awkwardly through a list of users to even see or access the "join server" button and expands the footer to over half of the monitor on desktop.
> 2. The discord icon does not link to the server, it links directly to discords website which users will most likely click on if they don't scroll.
> 3. Do users need to see the voice channels and users in the discord on the page before clicking join server?
> 
> If you insist on keeping the discord widget I would almost be more excited to create a customized version of it than using the default widget layout with something like the one below
> 
> 
> Just because something "works and everything is loading fine" does not mean its the best option.
> 
> If you want to keep everything as is that's ok just edit my PR or close it I'll work on a custom widget if it is a must have when I have time.

It's not that "_Just because something "works and everything is loading fine" does not mean its the best option._"
You are pointing out some valid critiques _now_, but they were not in your initial PR where the main reason for replacement are CORS errors which are IMO harmless as they are thrown only for default avatars. But you were not providing an apples-to-apples replacement, that's why I've asked it to be reverted as losing functionality for some harmless errors is not a solution.

My first glance comparison was:

**shields.io**:
Pros:
* Simple
* Users count

Cons:
* **No call to action**
* Styling is not DIscord's
* No list of online users


**Discord** widget:
Pros:
* Users count
* Styling distinctive to Discord

when expanded:
* List of online users
* Call to action
* Voice channels (opinable)

Cons:
* Animation on expansion (fixable)
* Voice channels (opinable)
* Call to action only visible when expanded

No **call to action** in your shields.io implementation is a major stopper for me (it's also why I didn't like shields.io for _X_). As for other _pros_ missing, I can deal with it if for others it's ok.

lyricalcasanova commented:
<details>
<summary>Previous Replies</summary>
> > > > I'm confused... the reason for this PR mainly was to fix the issue originally opened #914 unless maybe you know a way to fix the CORS error without removing the iFrame / making sure no one has a default profile picture / avatar and finding a way to fix the discord widget from extending off the page, I am all for it but currently there is no known resolutions hence why I said in the OP above: (I did search for others who had this issue but it seems like the widget for discord was made in 2016 and the most recent posts I found were in 2022 with no answers that are similar to this)
> > > > > Not sure why the default profile avatars for discord are throwing CORS errors so I assume it's best to just remove it / replace it (Also when hovered it extended past the bottom of the screen)
> > > 
> > > 
> > > As of 15 October 2023 they are still actively supporting the widget: [discord/discord-api-docs#6447](https://github.com/discord/discord-api-docs/issues/6447) I sincerely do not understand why you think the widget is bad?
> > > Its expanding and changing footer height with animation is not ideal, but that can be solved with layout updates, not by removing the widget.
> > 
> > 
> > I can give a list of reasons why I don't think the discord widget is good
> > 
> > 1. It expands and causes weird issues where users have to scroll awkwardly through a list of users to even see or access the "join server" button and expands the footer to over half of the monitor on desktop.
> > 2. The discord icon does not link to the server, it links directly to discords website which users will most likely click on if they don't scroll.
> > 3. Do users need to see the voice channels and users in the discord on the page before clicking join server?
> > 
> > If you insist on keeping the discord widget I would almost be more excited to create a customized version of it than using the default widget layout with something like the one below
> > Just because something "works and everything is loading fine" does not mean its the best option.
> > If you want to keep everything as is that's ok just edit my PR or close it I'll work on a custom widget if it is a must have when I have time.

</details>

> 
> It's not that "_Just because something "works and everything is loading fine" does not mean its the best option._" You are pointing out some valid critiques _now_, but they were not in your initial PR where the main reason for replacement are CORS errors which are IMO harmless as they are thrown only for default avatars and the rest works fine. But you were not providing an apples-to-apples replacement, that's why I've asked it to be reverted as losing functionality for some harmless errors is not a solution.
> 
> My first glance comparison was:
> 
> **shields.io**: Pros:
> 
> * Simple
> * Users count
> 
> Cons:
> 
> * **No call to action**

I find the **Discord 300 Online** more of a *call to action* than a generic discord widget, that users get lost just trying to find the "join server" button because it overflows off the screen then users have to scroll down users list to the bottom battling the on hover transition.
Not to mention we have a true call to action already in the footer...
![image](https://github.com/the-hideout/tarkov-dev/assets/33468668/516e8b60-7810-42c1-a16c-23170b9c6e97)

> * Styling is not DIscord's

Styling in Shields is subjective, I simply used the color pallet that I thought looks clean
if needed discord colors can be used for that also

> * No list of online users

Do you **really** need to see the online users list in the discord before you join? 
I can't remember a single time I checked a user list to see if I wanted to join a discord...

> 
> **Discord** widget: Pros:
> 
> * Users count

User count - **Online Users** is also in Shields.io as one of your pros above
![image](https://github.com/the-hideout/tarkov-dev/assets/33468668/029fcaf3-cd33-4d15-ac58-ce47dfc3d12a)

> * Styling distinctive to Discord

**See above** (Unless you mean the widget itself)

> 
> when expanded:
> 
> * List of online users

**See above**

> * Call to action

**See above**

> * Voice channels (opinable)

Same argument as the users list I have never worried about how many users are in the voice channels especially for a server centered around developing and tarkov news/updates
(This would be different if the server was centered around just gaming)

> 
> Cons:
> 
> * Animation on expansion (fixable)
> * Voice channels (opinable)
> * Call to action only visible when expanded
> 
> No **call to action** in your shields.io implementation is a major stopper for me (it's also why I didn't like shields.io for _X_). As for other _pros_ missing, I can deal with it if for others it's ok.

With all respect, @Shebuka it seems we can't come to a solid agreement. You do have the option to contribute your own code / changes that you would like to see if you feel strongly about anything. From the consensus so far, it seems like you may be the only one not okay with removing this generic widget 
I would understand your argument if you created the code being removed, but it's simply a widget that can always be moved to somewhere else or added later if a better option is presented.

I'm tired of debating over 32 changes, at this point unless there is truly a valid argument to make I will leave it up to the maintainers to decide what they want to do with this PR either Close, Edit, or Merge.

Shebuka commented:
Oh man, just wow... In the last comment, I've explained to you the reason behind my INITIAL request to rollback... I thought it was clear with using the past tense...

> Not to mention we have a true call to action already in the footer...

Where is a call to action here? (your screen)
![image](https://github.com/the-hideout/tarkov-dev/assets/33468668/78db8f01-7d86-43cb-b6b3-2e5ad1f3234c)

YOU said that 

> I'll work on a custom widget

and Im saying _ok if it has a call to action and is stylized like DIscord_ as YOU showed in the screenshot.

![image](https://github.com/the-hideout/tarkov-dev/assets/33468668/4519a0a3-9ba7-4bf4-b0fb-0e24ab6c907a)


Where does even

> With all respect, @Shebuka it seems we can't come to a solid agreement.

come from???

Razzmatazzz commented:
Seems like this discussion is causing some frustration for multiple people. Let's not lose sight of the fact that we all have the common goal of making the website as good as it can be to be as useful to the community as possible. Yes, we will sometimes have differences of opinion on what makes the website the best, but we should be able to discuss those differences respectfully and reach a consensus on the best way to move forward.

My two cents is that the current discord widget causes some annoying behavior (particularly on mobile). The shields.io solution seems like a good substitute as it provides a recognizable join link plus the current user count. While it does lack the ability to see who is online, voice channels, etc., I don't view those features as being vital (especially when crammed into the website footer).

lyricalcasanova commented:
> Oh man, just wow... In the last comment, I've explained to you the reason behind my INITIAL request to rollback... I thought it was clear with using the past tense...
> 
> > Not to mention we have a true call to action already in the footer...
> 
> Where is a call to action here? (your screen) ![image](https://private-user-images.githubusercontent.com/33468668/320191795-78db8f01-7d86-43cb-b6b3-2e5ad1f3234c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NDk3MzEsIm5iZiI6MTcxMjc0OTQzMSwicGF0aCI6Ii8zMzQ2ODY2OC8zMjAxOTE3OTUtNzhkYjhmMDEtN2Q4Ni00M2NiLWI2YjMtMmU1YWQxZjMyMzRjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDExNDM1MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdhMjc3NmMzZjA1MDI0YzI5YjljNGRmZDcwMGQ4N2ZlNjcyYTc1YTA0MmQ2OTU2ZWE2M2RkOWNkMWJjMWM0NzQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.8c3R-ftJFqxqorcbtoht9YCqQhHd83rm8cA1ee9-G_o)
> 
> YOU said that
> 
> > I'll work on a custom widget
> 
> and Im saying _ok if it has a call to action and is stylized like DIscord_ as YOU showed in the screenshot.
> 
> ![image](https://private-user-images.githubusercontent.com/33468668/320545118-4519a0a3-9ba7-4bf4-b0fb-0e24ab6c907a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NDk3MzEsIm5iZiI6MTcxMjc0OTQzMSwicGF0aCI6Ii8zMzQ2ODY2OC8zMjA1NDUxMTgtNDUxOWEwYTMtOWJhNy00YmY0LWIwZmItMGUyNGFiNmM5MDdhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDExNDM1MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWUxZDUwZmUxYmE1N2MwYmZjOTc0ZDlkMzJiNWM0NDcwMTk5NDgzYWRkN2FhMzYzOGRhMTZmOTQ3MTczMjU0ZDQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.2RfO_Pd9Zy8mkUgRKmAlJ3jDOgngv14KsGxIvax05bc)
> 
> Where does even
> 
> > With all respect, @Shebuka it seems we can't come to a solid agreement.
> 
> come from???

---

The call to action is the shields.io with a count of the online users, this causes curiosity from the users to wonder what is in the discord which makes them want to click it.

The requests to rollback was done as you requested with the exception of the discord widget as you did not mention the widget in your original request here:

> Discord: Simplified to just use a shields.io img/button that shows users online count (Still requires widget to be enabled on server)
> X/Twitter: Also changed to use shields.io img/button to view X/Twitter

These are some significant changes and I would like to keep them as they were for now, revert please.

> Removed the " : " in the "Made with love by"

The use of a colon here is justified as it introduces a list of contributors, so please revert it back too :)

---

Maybe there was some misunderstanding from the above comment but I see no reference in this comment anywhere specifying the discord widget not being removed simply just asking "revert please." with my change points referenced about the shields.io for x/twitter and discord this could not be further from clearly communicating in the future please provide more details, examples, or code so it's more clear.

I did as you requested which is the current state of this PR, then you asked where the widget is? 
I told you I could make one later on, if required or you can contribute also.
You then continued to argue that its not valid to remove the widget, so basically should I not contribute and revert all the changes?
What would make you happy?
I'm a bit confused, maybe if you propose some code or a demonstration of what your changes would look like then I would understand better I would love to see what your final product looks like and how it functions compared to what I proposed.

lyricalcasanova commented:
Also please enlighten me maybe my understanding of a "Call To Action" is wrong I thought a call to action was something that provokes people to click or find interest in doing something like a button / link? Is the button that says "Discord 283 Online" with a link to the Discord Server not a Call To Action?

Maybe it has to be a big bright button that follows none of the websites color scheme and jumps off the page or says "CLICK TO JOIN THE DISCORD" but I'm not sure because "Call To Action" in website design is defined as: `A prompt on a website that tells the user to take some specified action. A call to action is typically written as a command or action phrase, such as 'Sign Up' or 'Buy Now' and generally takes the form of a button or hyperlink.`

lyricalcasanova commented:
*Note: Accidentally merged the wrong branch (#911) force pushed to revert the merge, I would normally just add a revert commit but this was necessary*

@Shebuka and @Razzmatazzz 
I misunderstood what was exactly wanted / expected, but I feel like I was misunderstood also.

- My intentions were not to just fully remove the widget forever, I just did not want to rush another solution and make this PR even more difficult.
- The goal was a simple edit + remove the widget to keep console logs clean (and close #914) while working on a suitable replacement to be added in a later PR separating concerns.
- When I sent the example of the customized widget, it was simply an image for validation / reference. I have not currently made a replacement, but checked the code examples to ensure it was possible. (This most likely seemed like I had a ready to go, drop in replacement due to the example) **I realize now I didn't even explain this so I see the frustration here.**
- What Confused Me: The section in the footer reverted back: `If you wanna have a chat, ask questions or request features, we have a  Discord server.` satisfies the need of having a stable reference / call to action without the widget to access the discord. (Hyperlink to the discord server)
- Moving forward, if possible we should be a little bit more precise about what is wanted, expected, or planned to avoid anymore confusion or miscommunication so we can focus on developing meaningful changes and less on wasting time. (This goes for myself as well, since this is my 1st/2nd PR ever let alone working with others on this project specifically so I appreciate the patience but need the clarity)

**I have reverted the change and included the Discord Widget until there's an acceptable replacement for it as Shebuka is right it's best to have both options until then.**

Razzmatazzz commented:
.deploy to development

Shebuka commented:
.deploy
